athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/examples/ExampleMetadataHandlerTest.java

```diff
@@ -112,7 +113,8 @@ public void doListSchemas()
     public void doListTables()
     {
         logger.info("doListTables - enter");
-        ListTablesRequest req = new ListTablesRequest(IdentityUtil.fakeIdentity(), "queryId", "default", null);
+        ListTablesRequest req = new ListTablesRequest(IdentityUtil.fakeIdentity(),
+                "queryId", "default", null, null, NULL_PAGE_SIZE);
```

Comment:
is this a useful test? wouldn't it be better to have a real page size and also pass a new next tokens and test pagination?

Reply:
Added pagination to ExampleMetadataHandler.doListTables implementation and added tests to ExampleMetadataHandlerTest class.